BinaryEncoding.md

```diff
@@ -183,7 +183,7 @@ The module starts with a preamble of two fields:
 | Field | Type | Description |
 | ----- |  ----- | ----- |
 | magic number | `uint32` |  Magic number `0x6d736100` (i.e., '\0asm') |
-| version | `uint32` | Version number, currently 0xd. The version for MVP will be reset to 1. |
+| version | `uint32` | Version number, `0x1` as of the MVP |
```

Comment:
It seems like "as of the MVP" will become decreasingly relevant and may end up being ambiguous as to whether the version number is expected to change, so I'd remove it.